multicluster/test/e2e/antreapolicy_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+
+	antreae2e "antrea.io/antrea/test/e2e"
+)
+
+const (
+	// Provide enough time for policies to be enforced & deleted by the CNI plugin.
+	networkPolicyDelay          = 2000 * time.Millisecond
```

Comment:
maybe change it to `2 * time.Second` which is more readable.

Reply:
Done